app/models/salt_handler/minion_orchestration.rb

```diff
@@ -12,16 +12,18 @@ def self.can_handle_event?(event)
     parsed_event_data = JSON.parse event.data
     parsed_event_data["fun"] == "runner.state.orchestrate" &&
       (parsed_event_data["fun_args"].first == "orch.kubernetes" ||
-       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes")
+       parsed_event_data["fun_args"].first == "orch.update" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.kubernetes" ||
+       parsed_event_data["fun_args"].first["mods"] == "orch.update")
```

Comment:
To avoid repetition, maybe you can create a helper method that checks "fun_args". So you can call it:

```ruby
salt_args_match(parsed_event_data, "orch.kubernetes", "orch.update")

...

def salt_args_match(data, *values)
  d = data.first
  return if d.nil?

  values.each { |v| return true if d == v || d["mods"] == v }
  false
end
```